File: soma1bi.cs
using System;
using System.Diagnostics;

class Program
{
    static void Main()
    {
        long tot = 0;
        long n = 1_000_000_000;
        Stopwatch stopwatch = new Stopwatch();
        stopwatch.Start();
        for (long i = 1; i <= n; i++){tot += i;}
        stopwatch.Stop();
        double elapsedSeconds = (double)stopwatch.ElapsedMilliseconds / 1000.0;
        Console.WriteLine("Soma: " + tot);
        Console.WriteLine("Tempo: " + elapsedSeconds.ToString("F15"));
    }
}

File: soma1bi.rs
use std::time::{Instant};

fn main() {
    let mut tot: u64 = 0;
    let n: u64 = 1_000_000_000;
    let ti = Instant::now();
    for i in 1..=n {tot += i;}
    let tf = Instant::now();
    let dt = tf.duration_since(ti);
    println!("Soma: {}", tot);
    println!("Tempo: {:.15}", dt.as_secs_f64());
}